.github/workflows/ci.yml

```diff
@@ -256,6 +256,7 @@ jobs:
       - run: python -m pytest -n 2 modin/test/test_partition_api.py
       - run: python -m pytest modin/test/test_utils.py
       - run: python -m pytest asv_bench/test/test_utils.py
+      - run: python -m pytest modin/test/exchange_protocol/base
```

Comment:
The most convenient location for the tests to be placed would be near the protocol's code (`modin/core/dataframe/base/dataframe/protocol/tests`). However, this location creates import conflicts between `conftest.py` and `modin.utils`, in particular, this assertion is failing:
https://github.com/modin-project/modin/blob/dae9891dd4f25faf74dc411770d69cccc9ea2fad/modin/conftest.py#L24-L26

So it was decided to move the tests to a higher level.

Reply:
That is okay. We actually want to move all test in one place as part of #3645. However, I would like to generalize the structure a bit. See latest commit.

Reply:
In order to be ready for adding other exchange protocols.

Reply:
Agree with @YarShev.